fitbenchmarking/fitting_benchmarking.py

```diff
@@ -222,9 +225,14 @@ def make_plots(prob, group_results_dir, best_fit, wks, previous_name, count, use
     @param count :: number of different starting points for one problem
     @param user_func :: fitting function
     '''
+    if "neutron" in visuals_dir:
+        VDPage_dir = os.path.join(visuals_dir, "tables", "support_pages")
```

Comment:
please change VDPage_dir to support_page_dir. Please always use PEP 8 style variable names